golden_test/demo_test.dart

```diff
@@ -0,0 +1,85 @@
+// Copyright 2019 The Flutter team. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'package:flutter/material.dart';
+import 'package:flutter_test/flutter_test.dart';
+
+import 'package:gallery/main.dart';
+
+import 'testing/precache_images.dart';
+import 'testing/util.dart';
+
+const demoBannerRoute = '/demo/banner';
+
+void main() {
```

Comment:
As discussed offline, rename all these tests to not specify banner but instead be more generic for demo page

Reply:
Done!